rust/src/lockfile.rs

```diff
@@ -197,13 +224,25 @@ mod ffi {
     pub extern "C" fn ror_lockfile_write(
         filename: *const libc::c_char,
         packages: *mut glib_sys::GPtrArray,
+        rpmmd_repos: *mut glib_sys::GPtrArray,
         gerror: *mut *mut glib_sys::GError,
     ) -> libc::c_int {
         let filename = ffi_view_os_str(filename);
         let packages: Vec<*mut DnfPackage> = ffi_ptr_array_to_vec(packages);
+        let rpmmd_repos: Vec<*mut DnfRepo> = ffi_ptr_array_to_vec(rpmmd_repos);
+
+        // get current time, but scrub nanoseconds; it's overkill to serialize that
```

Comment:
What if we instead something like reference the last time the file changed in git?  Dunno.

Reply:
Heh yeah, that's in fact exactly what I initially did (see the commit message in https://github.com/coreos/fedora-coreos-releng-automation/commit/c932c34c4db87979f141178c3e760d626b46c214). Though there were concerns that it was too magic/implicit.